### PR TITLE
Add MANIFEST.in

### PR DESCRIPTION
**Pull request by fabaff**
Ensure that the source released on PyPI contains the LICENSE file.

Shipping the license file is a requirement for various distributions.

### CODE REVIEW

**Comment by fabaff** [comment]
Would be nice to have a new release after merging this.

File: MANIFEST.in [hunk 1]
```diff
@@ -0,0 +1 @@
+include LICENSE
```
Comment (MartinHjelmare) [inline MANIFEST.in]: It seems we have two license files in the source. Which one do we want to publish?

Comment (fabaff) [reply MANIFEST.in]: `LICENSE` was first commited. `LICENSE.md` has a different `Copyright` line which is a bit off in regard of the year.

Comment (MartinHjelmare) [reply MANIFEST.in]: We want to keep the markdown file.
```suggestion
include LICENSE.md
```

**Review by MartinHjelmare — APPROVED** [review]
Thanks!

**Pull request merged**